config.toml

```diff
@@ -0,0 +1,53 @@
+## hop_type options are "mock" or "hop"
```

Comment:
I agree that refactoring the directory structure is desirable; in fact, I've considered that it might be better to have an entirely separate repo that defines this Compose configuration, where the initialization step would include git cloning the involved repos, including archive-core, archive-api, and archive-ingest.